backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -706,627 +591,1031 @@ describe('ContributionResolver', () => {
+          })
+          expect(contributionListResult).toMatchObject({
+            contributionCount: 6,
+            contributionList: expect.arrayContaining([
+              {
+                amount: '100',
+                id: contributionToConfirm.data.createContribution.id,
+                memo: 'Test contribution to confirm',
               },
-            },
-          }),
-        )
+              {
+                id: pendingContribution.data.createContribution.id,
+                memo: 'Test PENDING contribution update',
+                amount: '10',
+              },
+              {
+                id: contributionToDeny.data.createContribution.id,
+                memo: 'Test contribution to deny',
+                amount: '100',
+              },
+              {
+                id: contributionToDelete.data.createContribution.id,
+                memo: 'Test contribution to delete',
+                amount: '100',
+              },
+              {
+                id: inProgressContribution.data.createContribution.id,
+                memo: 'Test IN_PROGRESS contribution',
+                amount: '100',
+              },
+              {
+                id: bibiCreatedContribution.id,
+                memo: 'Herzlich Willkommen bei Gradido!',
+                amount: '1000',
+              },
+            ]),
+          })
+        })
       })
 
-      it('returns all CONFIRMED and PENDING Creation', async () => {
-        await expect(
-          query({
-            query: listAllContributions,
+      describe('filter confirmed is true', () => {
+        it('returns only unconfirmed creations', async () => {
+          const {
+            data: { listContributions: contributionListResult },
+          }: { data: { listContributions: ContributionListResult } } = await query({
+            query: listContributions,
             variables: {
               currentPage: 1,
               pageSize: 25,
               order: 'DESC',
-              statusFilter: ['CONFIRMED', 'PENDING'],
-            },
-          }),
-        ).resolves.toEqual(
-          expect.objectContaining({
-            data: {
-              listAllContributions: {
-                contributionCount: 2,
-                contributionList: expect.arrayContaining([
-                  expect.objectContaining({
-                    id: expect.any(Number),
-                    state: 'CONFIRMED',
-                    memo: 'Herzlich Willkommen bei Gradido!',
-                    amount: '1000',
-                  }),
-                  expect.objectContaining({
-                    id: expect.any(Number),
-                    state: 'PENDING',
-                    memo: 'Test env contribution',
-                    amount: '100',
-                  }),
-                ]),
-              },
+              filterConfirmed: true,
             },
-          }),
-        )
+          })
+          expect(contributionListResult).toMatchObject({
```

Comment:
`expect(contributionListResult.contributionList).toHaveLength(4)`
and remove the `not`

Reply:
I mean to add an additional expect, this one and the one you have without the nots

Reply:
Added